diff --git a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
index c259beb..6990a05 100644
--- a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
+++ b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
@@ -107,7 +107,7 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
         as={RouterNavLink}
         workplace={formatMessage({
           id: 'app.components.LeftMenu.navbrand.workplace',
-          defaultMessage: 'Workplace',
+          defaultMessage: 'Dashboard',
         })}
         title={menuTitle}
         icon={
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
index 8fb94c5..295fcd9 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
@@ -19,44 +19,15 @@ const HomeHeader = ({ hasCreatedContentType, onCreateCT }) => {
       <Box paddingLeft={6} paddingBottom={10}>
         <Flex direction="column" alignItems="flex-start" gap={5}>
           <Typography as="h1" variant="alpha">
-            {hasCreatedContentType
-              ? formatMessage({
-                  id: 'app.components.HomePage.welcome.again',
-                  defaultMessage: 'Welcome 👋',
-                })
-              : formatMessage({
-                  id: 'app.components.HomePage.welcome',
-                  defaultMessage: 'Welcome on board!',
-                })}
+            Root Games
           </Typography>
           <WordWrap textColor="neutral600" variant="epsilon">
-            {hasCreatedContentType
-              ? formatMessage({
-                  id: 'app.components.HomePage.welcomeBlock.content.again',
-                  defaultMessage:
-                    'We hope you are making progress on your project! Feel free to read the latest news about Strapi. We are giving our best to improve the product based on your feedback.',
-                })
-              : formatMessage({
-                  id: 'app.components.HomePage.welcomeBlock.content',
-                  defaultMessage:
-                    'Congrats! You are logged as the first administrator. To discover the powerful features provided by Strapi, we recommend you to create your first Content type!',
-                })}
+            Welcome to Root Games Admin! Add more games, categories and publishers from Content Manager.
           </WordWrap>
-          {hasCreatedContentType ? (
-            <Link isExternal href="https://strapi.io/blog">
-              {formatMessage({
-                id: 'app.components.HomePage.button.blog',
-                defaultMessage: 'See more on the blog',
-              })}
-            </Link>
-          ) : (
-            <Button size="L" onClick={onCreateCT} endIcon={<ArrowRight />}>
-              {formatMessage({
-                id: 'app.components.HomePage.create',
-                defaultMessage: 'Create your first Content type',
-              })}
-            </Button>
-          )}
+
+          <Link isExternal href="https://aqua9.com.br">
+            Let's go to Aqua9
+          </Link>
         </Flex>
       </Box>
     </div>
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
index e290ab2..9f6a857 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
@@ -93,21 +93,6 @@ const socialLinks = [
     icon: <StyledDiscourse />,
     alt: 'forum',
   },
-  {
-    name: { id: 'app.components.HomePage.community.links.blog', defaultMessage: 'Blog' },
-    link: 'https://strapi.io/blog?utm_source=referral&utm_medium=admin&utm_campaign=career%20page',
-    icon: <StyledStrapi />,
-    alt: 'blog',
-  },
-  {
-    name: {
-      id: 'app.components.HomePage.community.links.career',
-      defaultMessage: 'We are hiring!',
-    },
-    link: 'https://strapi.io/careers?utm_source=referral&utm_medium=admin&utm_campaign=blog',
-    icon: <StyledStrapi />,
-    alt: 'career',
-  },
 ];
 
 const LinkCustom = styled(LinkButton)`
@@ -136,16 +121,7 @@ const SocialLinks = () => {
 
   const socialLinksExtended = [
     ...socialLinks,
-    {
-      icon: <StyledStrapi />,
-      link: communityEdition
-        ? 'https://discord.strapi.io'
-        : 'https://support.strapi.io/support/home',
-      name: {
-        id: 'Settings.application.get-help',
-        defaultMessage: 'Get help',
-      },
-    },
+
   ];
 
   return (
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
index a22f746..6c88177 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
@@ -76,9 +76,6 @@ export const HomePageCE = () => {
             </GridItem>
           </Grid>
           <Grid gap={6}>
-            <GridItem col={8} s={12}>
-              {showGuidedTour ? <GuidedTourHomepage /> : <ContentBlocks />}
-            </GridItem>
             <GridItem col={4} s={12}>
               <SocialLinks />
             </GridItem>
